pages/docs/balance/api-reference/open-banking/list-available-exchange-connections.mdx

```diff
@@ -0,0 +1,106 @@
+---
+layout: guides
+product: balance
+title: "List available exchange connections for a customer"
+guide:
+  step: 3
```

Comment:
```suggestion
  step: 4
```

Let's update the step of this page to be `4` and that of "retrieve-bank-balance.mdx" to be `5`.